### PR TITLE
Fix/rkt 7601 es5

### PR DESCRIPTION
**Pull request by robertsevernsentient**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
```
Comment (mattstrom) [inline .eslintrc.json:4]: What role does this play?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "plugins": ["es"],
+  "extends": [
+    "eslint:recommended"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
```
Comment (mattstrom) [inline .eslintrc.json:15]: And this?

File: package.json [hunk 3]
```diff
@@ -49,6 +50,8 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "chai-spies": "^1.0.0",
+    "eslint": "^7.4.0",
+    "eslint-plugin-es": "^3.0.1",
     "jsdoc": "^3.6.3",
     "mocha": "^7.1.0",
     "webcrypto": "git+https://github.com/nodejs/webcrypto.git",
```
Comment (mattstrom) [inline package.json:57]: I would recommend adding [husky](https://www.npmjs.com/package/husky) as well for pre-commit hooks.

File: .eslintrc.json (hunk 1 above)
Comment (robertsevernsentient) [reply .eslintrc.json:4]: It allows Set and Map

File: .eslintrc.json (hunk 2 above)
Comment (robertsevernsentient) [reply .eslintrc.json:15]: globals aren't needed - removing
plugins has the extra es6 specific rules
extends - the basic linting rules that eslint uses as standard
parserOptions - was in the base example

Comment (robertsevernsentient) [reply .eslintrc.json:15]: ecmaVersion removed

File: src/beacon.js [hunk 4]
```diff
@@ -73,6 +73,7 @@ export default function Emitter(endpoint, context) {
         }
       });
     } else {
+      // eslint-disable-next-line no-constant-condition
```
Comment (mattstrom) [inline src/beacon.js:76]: 🤔

Comment (robertsevernsentient) [reply src/beacon.js:76]: You can argue for changing the approach on this -- it escapes with a break.

**Pull request closed without merging**